resource-managers/kubernetes/integration-tests/pom.xml

```diff
@@ -220,6 +220,26 @@
               </resources>
             </configuration>
           </execution>
+          <execution>
+              <id>copy-integration-python</id>
+              <phase>pre-integration-test</phase>
+              <goals>
+                  <goal>copy-resources</goal>
+              </goals>
+              <configuration>
+                  <outputDirectory>${project.build.directory}/docker/python</outputDirectory>
+                  <resources>
+                      <resource>
+                          <directory>${project.parent.basedir}/python</directory>
+                          <filtering>true</filtering>
+                          <excludes>
+                            <exclude>${project.parent.basedir}/python/.egg</exclude>
+                            <exclude>${project.parent.basedir}/python/dist</exclude>
+                          </excludes>
+                      </resource>
+                  </resources>
+              </configuration>
+          </execution>
```

Comment:
This part is what is incomplete for making fully fledged integration tests. It seems that we need to mimic the environment created by the make_distribution.sh environment, done with the following script:
```bash
# Make pip package
if [ "$MAKE_PIP" == "true" ]; then
  echo "Building python distribution package"
  pushd "$SPARK_HOME/python" > /dev/null
  python setup.py sdist
  popd > /dev/null
else
  echo "Skipping building python distribution package"
fi
```
Same environment will need to be mimicked for testing R bindings. Any recommendations would be great :)